topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -0,0 +1,321 @@
+
+> ### {% icon hands_on %} Hands-on: Run JBrowse
+>
+> 1. **JBrowse** {% icon tool %} with the following parameters
+>    - *"Reference genome to display"*: `Use a genome from history`
+>       - *"Select the reference genome"*: `https://zenodo.org/record/3497110/files/MTB_ancestor_reference.fasta`
+>
+>       This sequence will be the reference against which annotations are displayed
+>
+>    - *"Produce Standalone Instance"*: `Yes`
+>    - *"Genetic Code"*: `11: The Bacterial, Archaeal and Plant Plastid Code`
+>    - *"JBrowse-in-Galaxy Action"*: `New JBrowse Instance`
+>    - *"Track Group"*
+>
+>        We will now set up three different tracks - these are datasets displayed underneath the reference sequence (which is displayed as nucleotides in FASTA format). We will choose to display the sequence reads (the .bam file), the variants found by snippy (the .gff file) and the annotated reference genome (the wildtype.gff)
+>
+>       - **Track 1 - sequence reads**: Click on `Insert Track Group` and fill it with
+>           - "Track Category" to `sequence reads`
+>           - Click on `Insert Annotation Track` and fill it with
+>               - "Track Type" to `BAM Pileups`
+>               - "BAM Track Data" to `snippy on data XX, data XX, and data XX mapped reads (bam)`
+>               - "Autogenerate SNP Track" to `Yes`
+>               - "Track Visibility" to `On for new users`
+>       - **Track 2 - variants**: Click on `Insert Track Group` and fill it with
+>           - "Track Category" to `variants`
+>           - Click on `Insert Annotation Track` and fill it with
+>               - "Track Type" to `VCF SNPs`
+>               - "SNP Track Data" to `TB Variant Filter on data XX`
+>               - "Track Visibility" to `On for new users`
+>       - **Track 3 - annotated reference**: Click on `Insert Track Group` and fill it with
+>           - "Track Category" to `annotated reference`
+>           - Click on `Insert Annotation Track` and fill it with
+>               - "Track Type" to `GFF/GFF3/BED/GBK Features`
+>               - "GFF/GFF3/BED Track Data" to `https://zenodo.org/record/3531703/files/Mycobacterium_tuberculosis_h37rv.ASM19595v2.45.chromosome.Chromosome.gff3`
+>               - "JBrowse Track Type [Advanced]" to `Canvas Features`
+>               - Click on "JBrowse Styling Options [Advanced]"
+>               - "JBrowse style.label" to `product`
+>               - "JBrowse style.description" to `product`
+>               - "Track Visibility" to `On for new users`
+{: .hands_on}
+
+A new dataset will be created in your history, containing the JBrowse interactive visualisation. We will now view its contents and play with it by clicking the {% icon galaxy-eye %} (eye) icon of the `JBrowse on data XX and data XX - Complete` dataset. The JBrowse window will appear in the centre Galaxy panel.
```

Comment:
You can actually embed jbrowse in here if it's instructive/useful enough? I've done that in another PR. At least a screenshot might be good here though.

Reply:
Can you point me to an example of how you embedded JBrowse? I would like to include an image but am still considering the best way to arrange it.

Reply:
https://github.com/galaxyproject/training-material/pull/1547 has an example. I've made a PR to the jbrowse tool which enables outputting those. We could consider adding one jbrowse to the GTN and then re-using it with all of the different tutorial data directories as well.